core/commands.inc.sh

```diff
@@ -45,30 +43,6 @@ do_configure() {
   builder_finish_action success configure:$target
 }
 
-has_configured_dependencies=false
-
-do_configure_dependencies() {
```

Comment:
This is no longer needed, because it is handled by the dependency refs in build.sh. We can assume node is present for test actions, so we don't need to skip those deps now either.